base/docs/helpdb.jl

```diff
@@ -11559,6 +11559,13 @@ Checkout the `Pkg.dir(pkg)` repo to the branch `branch`. Defaults to checking ou
 Pkg.checkout(pkg)
 
 doc"""
+    setprotocol!(proto)
```

Comment:
it would be better to put this docstring at the function definition rather than helpdb (we're trying to empty out this file).